cherrypy/tutorial/tut06_default_method.py

```diff
@@ -22,12 +22,16 @@
 
 
 class UsersPage:
+    """The UsersPage Class for Tutorial 06."""
 
     @cherrypy.expose
     def index(self):
-        # Since this is just a stupid little example, we'll simply
-        # display a list of links to random, made-up users. In a real
-        # application, this could be generated from a database result set.
+        """Index for UsersPage.
+
+        Since this is just a stupid little example, we'll simply
+        display a list of links to random, made-up users. In a real
+        application, this could be generated from a database result set.
+        """
```

Comment:
```suggestion
        """Produce HTTP response body of the users app index URI."""
        # Since this is just a stupid little example, we'll simply
        # display a list of links to random, made-up users. In a real
        # application, this could be generated from a database result set.
```